### PR TITLE
Scale the rofi window on HiDPI. Better DPI detection.

### PR DESCRIPTION
**Pull request by twa022**
#59

### CODE REVIEW

**Comment by twa022** [comment]
I don't have a hidpi monitor to test, but it worked okay when I forced the scale on mine.

Would it be better to make copies of the themes for hidpi and override the theme rather than just overriding the width so all the theme properties expressed in px can be doubled?

**Comment by twa022** [comment]
I like #62 better as a solution

File: usr/lib/mate-hud/mate-hud [hunk 1]
```diff
@@ -220,16 +220,24 @@ def get_menu(menuKeys):
...
-            xfce_custom_dpi = Xfconf.Channel.new_with_property_base("xsettings", "/Xft").get_int("/DPI", 0)
-        except:
-            logging.debug('running XFCE, but xfconf gi repository not available')
-        if xfce_custom_dpi > 0:
-            dpi = xfce_custom_dpi
+    dpi = (width_dpi + height_dpi) / 2
+
+    # from pyxr: https://github.com/WhatNodyn/Pyxr (MIT license)
+    # Get the xresource db so we can read Xft.dpi
+    res_prop = display.Display().screen().root.get_full_property(Xatom.RESOURCE_MANAGER, Xatom.STRING)
```
Comment (flexiondotorg) [inline usr/lib/mate-hud/mate-hud:227]: Sadly `pyxr` is not packaged for Debian/Ubuntu or Fedora/RHEL

**Review by flexiondotorg — CHANGES_REQUESTED** [review]
Can you rework this to not require pyxr? Perhaps this is available via GDK?

**Pull request closed without merging**